lldb/include/lldb/Breakpoint/BreakpointLocation.h

```diff
@@ -129,14 +129,10 @@ class BreakpointLocation
   ///
   /// \param[in] condition
   ///    The condition expression to evaluate when the breakpoint is hit.
```

Comment:
You removed the word "expression" from the help in Breakpoint.h, it's confusing to have these two differ.

Reply:
I dropped it because the condition is more than just the expression text. Removed it here too for consistency. In all the user-facing strings we just use "breakpoint condition".